adb_graphics/default_specs.yml

```diff
@@ -609,6 +621,54 @@ ptmp: # Potential temperature
     ticks: 10
     unit: K
     wind: 10m
+ptyp: # Hourly total precipitation
+  sfc:
+    clevs: [0.002, 0.01, 0.05, 0.1, 0.25, 0.50, 0.75, 1.0, 2.0]
+    cmap: gist_ncar
+    colors: pcp_colors
+    contours:
+      csnow_sfc:
+        colors: blue
+        levels: [0, 1]
+        linewidths: 0.2
+      crain_sfc:
+        colors: green
+        levels: [0, 1]
+        linewidths: 0.2
+      cfrzr_sfc:
+        colors: purple
+        levels: [0, 1]
+        linewidths: 0.5
+      cicep_sfc:
+        colors: red
+        levels: [0, 1]
+        linewidths: 0.5
+    hatches:
+      crain_sfc:
+        colors: ['green']
```

Comment:
Can the colors be a string? Will we ever need to have multiples here? 

This change would likely need a change in line below...I'll leave a comment there, too.